src/lib/containers/AccessApprovalsTable.tsx

```diff
@@ -0,0 +1,142 @@
+      : ('ASC' as SortDirection)
+    setSort([{ field, direction }])
+    setIsDescending(!isDescending)
+  }
+  return (
+    <div className="AccessApprovalsTable bootstrap-4-backport">
+      <Typography style={{ padding: '12px' }} variant="headline3">
+        Status in Access Requirements
+      </Typography>
+      <Table striped borderless bordered={false}>
+        <thead className="access-approval-header">
+          <tr>
+            <th className="thead-cell remove-border">AR ID</th>
+            <th className="thead-cell remove-border">
+              Access Requirement Name
+            </th>
+            <th className="thead-cell remove-border">Submitter</th>
+            <th className="thead-cell remove-border">Status</th>
+            <th className="thead-cell remove-border">
+              Modified Date
+              <SortIcon
+                role="button"
+                style={{ float: 'right' }}
+                active={false}
+                direction={
+                  sort && sort[0].field === 'MODIFIED_ON'
+                    ? sort[0].direction === 'DESC'
+                      ? Direction.DESC
+                      : Direction.ASC
+                    : Direction.DESC
+                }
+                onClick={() =>
+                  onSort(AccessApprovalSortField.MODIFIED_ON, isDescending)
+                }
+              />
+            </th>
+            <th className="thead-cell remove-border">
+              Expires
+              <SortIcon
+                role="button"
+                style={{ float: 'right' }}
+                active={false}
+                direction={
+                  sort && sort[0].field === 'EXPIRED_ON'
+                    ? sort[0].direction === 'DESC'
+                      ? Direction.DESC
+                      : Direction.ASC
+                    : Direction.DESC
+                }
+                onClick={() =>
+                  onSort(AccessApprovalSortField.EXPIRED_ON, isDescending)
+                }
+              />
+            </th>
+          </tr>
+        </thead>
+        <tbody>
+          {accessApprovals.map(item => {
+            let modifiedOn = formatDate(moment(item.modifiedOn))
+            let expiredOn = formatDate(moment(item.expiredOn))
+            return (
+              <tr key={item.id}>
+                <td className="remove-border">
+                  <a
+                    href={`${PRODUCTION_ENDPOINT_CONFIG.PORTAL}#!AccessRequirement:AR_ID=${item.accessRequirementId}`}
+                  >
+                    {item.accessRequirementId}
+                  </a>
+                </td>
+                <td className="remove-border">{item.accessRequirementName}</td>
+                <td className="remove-border">
+                  <UserCard size={SMALL_USER_CARD} ownerId={item.submitterId} />
+                </td>
+                <td className="remove-border">
+                  {item.state[0] + item.state.slice(1).toLocaleLowerCase()}
```

Comment:
I think using lodash upperFirst is more readable, took me a second to realize why you were using .slice
```suggestion
                  {upperFirst(item.state.toLocaleLowerCase())}
```